scripts/load-test/collect-pod-logs/README.md

```diff
@@ -0,0 +1,36 @@
+# JIRA task - https://issues.redhat.com/browse/STONE-831 
+
+# The bash script is collect-openshift-pod-logs.sh
+
+
+# The Jira task description:
+
+Description
+===========
+Goal is to be able to look at was happening in the cluster pods logs before the cluster dies.
+
+Input of the tool would be:
+it would be running on the host logged to the StoneSoup cluster
+it will be provided a list of namespaces
+The tool would check all the pods in all the namespaces and would be following (tailing) logs to files.
+
+
+The script logic:
+================
+Log directory configuration
+Format:  ./collect-openshift-pod-logs.sh logs-${USER}-$(date +%Y-%m-%d)  
+if no parameter given, a default will be used for the log_dir - logs-${USER}-$(date +%Y-%m-%d)
+
+1. The script will run in the background process
+2. Per each namespace, it will tail logs of a created or existing pod in the background and store it in a distinct filename
+3. All log files will be stored under a main log_dir of type log_dir="logs-${USER}-$(date +%Y-%m-%d)"
+4. If the script runs in the forground, the user may stop the script by sending a Trap SIGINT (Ctrl+C) which will call cleanup_and_exit function and exit safely 
```

Comment:
typo: forground

Reply:
I Josef
forground is not a typo

Did you mean for something else?